R/rate0compare.R

```diff
@@ -0,0 +1,271 @@
+#' stratified  MH TEST
+#' @param Treatment treament indicator
+#' @param Response response indicator, 1 for responsder, 0 for non-responder
+#' @param TRT_ord treatment order. The first one is control arm and the second one is experiment arm. Default is NULL
+#' @param stratified Default is NULL for unstratified MN. If TRUE, stratified MN will be conducted
+#' @param Stratum stratum information
+#' @param delta delta under the null

```

Comment:
Let's refer this to document variables: https://style.tidyverse.org/documentation.html. For example

```suggestion
#' @param delta a numeric value to set the difference of two group under the null

```